### DISCUSSION TITLE
Hi, just updating you about my life

### DISCUSSION THREAD

**Discussion by baruchiro** [brafdlog/caspion#129] [General]
I was a little busy (same us you), now I returned from reserve duty (מילואים), so I will return to work in the next days.

If you need anything urgent please point about it, I will find a time.

(Is this kind of discussion is OK here? Is the category right?)

**Comment by brafdlog** [comment]
I will have some time in the coming two weeks.

I created a pr to fix something in sheets #127 

@baruchiro I will create the following prs based on one another so when you will have time you could review them one by one.

**Reply by baruchiro** [reply]
OK, just remind you, I started to work on the Sheets backend, but I have no progress there, so you may work there also. (Maybe that is the PRs you're talking about)

**Reply by brafdlog** [reply]
What are the issues you know about there that need to be solved?

**Reply by baruchiro** [reply]
#120

**Reply by baruchiro** [reply]
> I will have some time in the coming two weeks.
> 
> I created a pr to fix something in sheets #127
> 
> @baruchiro I will create the following prs based on one another so when you will have time you could review them one by one.

What else except #127?

Can I start to work on #120 or you are in the middle of something in that area?

**Comment by baruchiro** [comment]
OK I'm back (I hope)
